mtags/src/main/scala-2/scala/meta/internal/mtags/OnDemandSymbolIndex.scala

```diff
@@ -231,11 +234,33 @@ final class OnDemandSymbolIndex(
     val noExtension = toplevel.value.stripSuffix(".").stripSuffix("#")
     List(
       noExtension + ".scala",
-      noExtension + ".java",
-      "java.base/" + noExtension + ".java" // For Java 11.
+      noExtension + ".java"
     )
   }
 
+  private def modulePaths(toplevel: Symbol): List[String] = {
+    if (Properties.isJavaAtLeast("9")) {
+      val noExtension = toplevel.value.stripSuffix(".").stripSuffix("#")
+      val javaSymbol = noExtension.replace("/", ".")
+      try {
+        sourceJars.loadClass(javaSymbol).toList.flatMap { cls =>
+          // note(@tgodzik) Modules are only available in Java 9+, so we need to ivoke this reflectively
+          val module = cls.getClass().getMethod("getModule").invoke(cls)
```

Comment:
for unnamed module it will return null

Reply:
Added options everywhere to be on the safe side.